plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -50,8 +50,8 @@
 			woocommerce_wp_text_input(
 				array(
 					'id'                => '_stock',
-					'value'             => wc_stock_amount( $product_object->get_stock_quantity( 'edit' ) ),
-					'label'             => __( 'Stock quantity', 'woocommerce' ),
+					'value'             => wc_stock_amount( $product_object->get_stock_quantity( 'edit' ) ?? 1 ),
```

Comment:
I added the changes here instead of the constructor to ensure we won't have any unexpected side effects.